LICENSE

```diff
@@ -2,7 +2,7 @@
                            Version 2.0, January 2004
                         http://www.apache.org/licenses/
 
-   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
+   TERM AND CONDITION FOR USE, REPRODUCTION, AND DISTRIBUTION
```

Comment:
Indeed, i guess you are right. But if you can create a little more space before and after 'AND' i would be happy to merge your request.

Reply:
sure, i will

Reply:
excellent